### DISCUSSION TITLE
Add persistQueryClient for Vue

### DISCUSSION THREAD

**Discussion by louia** [TanStack/query#4597] [Ideas] [answered]
Hi, 

It's possible to provide all the stuff to make persistQueryClient compatible with Vue ?

All the [docs ](https://tanstack.com/query/v4/docs/plugins/persistQueryClient)are about React but no mention to Vue framework

**Comment by DamianOsipiuk** [comment, marked as answer]
Please take a look at this example https://github.com/TanStack/query/blob/main/examples/vue/persister/src/main.ts
